src/python/pants/backend/rust/lint/rustfmt/subsystem.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.option.option_types import ArgsListOption, SkipOption
+from pants.option.subsystem import Subsystem
+from pants.util.ordered_set import FrozenOrderedSet
+
+# TODO we want to provide possibility for customising input for example using --config
+SUPPORTED_RUSTFMT_ARGS: FrozenOrderedSet[str] = FrozenOrderedSet(())
+SUPPORTED_RUSTFMT_ARGS_AS_HELP = ", ".join([f"`{arg}`" for arg in SUPPORTED_RUSTFMT_ARGS])
```

Comment:
The other subsystem usually have a dedicated `config` option.

Also, if we don't support any arguments, why do we have this option?

Reply:
Basically because we want to have it in the future for sure. It is just hard for me to tell exactly which flags we want to support atm. 

I will add the config option

Reply:
Usually I wont code for the future, but in this case I have to explicitly diverge on the go way of doing fmt. Hence because I know I will support some flags I do not want to explicitly remove something I know I want to add at a later point

Reply:
Or just remove it for now and open a new issue for adding that future support. That issue can track the need for having such code in a future PR.